pyartcd/pyartcd/util.py

```diff
@@ -595,9 +595,13 @@ def mail_build_failure_owners(failed_builds: dict, doozer_working: str, mail_cli
                             f'{failure["message"]}\n' \
                             f'{container_log}'
 
+        # Send email to owners of failed image builds
+        # If art is the only owner of image (example for our ci golang builder images) send instead to our default automation email
+        owner = failure['owners']
+                if (failure['owners'] and failure['owners'] != ["aos-team-art@redhat.com"])
```

Comment:
Probably needs `\` at EOLs right?

Reply:
Ah you're right, thanks for catching that

Reply:
fixed now